Updates to 'what is Boundary?' (#3315)
hashicorp/boundary: website/content/docs/overview/what-is-boundary.mdx

---
layout: docs
page_title: What is Boundary?
description: An introduction to Boundary
---

# What is Boundary?

> HashiCorp Boundary is an agentless identity-aware proxy. Boundary is intended to simplify and secure enabling least-privileged access to cloud infrastructure.

Reviewer: I think "agentless identity-aware proxy" may be too technical for this page as its many people's first impressions of Boundary

> HashiCorp Boundary is an agentless identity-aware proxy. Boundary is intended to simplify and secure enabling least-privileged access to cloud infrastructure.

Reviewer 2: ```suggestion
HashiCorp Boundary is an agentless identity-aware proxy. Boundary simplifies and secures enabling least-privileged access to cloud infrastructure.
```

With Boundary you can:
- Enable single sign-on to target services and applications via external identity providers
- Provide just-in-time network access to private resources
- Enable passwordless access with dynamic credentials via [HashiCorp Vault](https://www.vaultproject.io/)
- Automate discovery of new target systems
- Record and manage privilege sessions
- Standardize your team's access workflow with a consistent experience for any type of infrastructure across any provider

> Get started [here](/boundary/tutorials) 

Reviewer 2: ```suggestion
Get started [here](/boundary/tutorials). 
```

- **User Authentication:** The user logs in with a trusted identity (based on the rules and policies) with a trust identity platform such as Azure Active Directory, Okta, Ping, or any other trust identity platforms supporting OpenID Connect.
- **Granular Authorization:** Boundary authenticates and authorizes users based on their roles and logical services, and tightly controls access and actions performed against systems.
- **User-selected dynamic catalogs**: The user selects their application or host from dynamic host catalogs.
- **Access:** Boundary streamlines connection to hosts by automating discovery and access configuration as workloads are deployed or changed.

## Which edition of Boundary is right for me?

> - **HCP Boundary**: a managed offering of Boundary with commercial-only use-cases features. Boundary's control plane is hosted by HashiCorp and customers have the option of running private workers within their environment.

Reviewer 2: ```suggestion
- **HCP Boundary**: a managed offering of Boundary with commercial features. HashiCorp hosts Boundary's control plane and you have the option of running private workers within your environment.
```

> - **HCP Boundary**: a managed offering of Boundary with commercial-only use-cases features. Boundary's control plane is hosted by HashiCorp and customers have the option of running private workers within their environment.

Reviewer 3: ...commercial-only use-cases **and** features

- **Boundary OSS**: an open source, self-managed version of Boundary.

> - **Boundary Enterprise**: a self-managed offering of Boundary with full feature parity to HCP Boundary.

Reviewer: I would put OSS last, feels strangely sandwiched

If you're not sure which edition is right for you, we recommend [**HCP Boundary**](https://developer.hashicorp.com/boundary/tutorials/hcp-getting-started) because it eliminates deployment operations.

> Regardless of which server edition of Boundary you are hosting, all editions use the same Desktop and CLI clients, which you can download [here](https://developer.hashicorp.com/boundary/downloads).

Reviewer: ```suggestion
Regardless of which edition of Boundary you are using, all editions use the same Desktop and CLI clients, which you can download [here](https://developer.hashicorp.com/boundary/downloads).
```

Reviewer 3: Isn't the OSS CLI binary different from the ENT/HCP binary?

Author: No, the OSS binary has all CLI commands across editions of Boundary.

> ## Getting Started

Reviewer 2: This Getting Started section feels redundant to the new "Get Started here" link that was added to the introduction. I think we should delete one of them.

Author: Will do

- **User Authentication:** The user logs in with a trusted identity (based on the rules and policies) with a trust identity platform such as Azure Active Directory, Okta, Ping, or any other trust identity platforms supporting OpenID Connect.
- **Granular Authorization:** Boundary authenticates and authorizes users based on their roles and logical services, and tightly controls access and actions performed against systems.
- **User-selected dynamic catalogs**: The user selects their application or host from dynamic host catalogs.
- **Access:** Boundary streamlines connection to hosts by automating discovery and access configuration as workloads are deployed or changed.

## Which edition of Boundary is right for me?

> - **HCP Boundary**: a managed offering of Boundary with commercial-only use-cases and features. Boundary's control plane is hosted by HashiCorp and customers have the option of running private workers within their environment.

Reviewer 2: ```suggestion
- **HCP Boundary**: a managed Boundary offering with commercial features. HashiCorp hosts Boundary's control plane and you have the option of running private workers within your environment.
```

> - **Boundary Enterprise**: a self-managed offering of Boundary with full feature parity to HCP Boundary.

Reviewer 2: ```suggestion
- **Boundary Enterprise**: a self-managed Boundary offering with full feature parity to HCP Boundary.
```

- **Boundary OSS**: an open source, self-managed version of Boundary.

If you're not sure which edition is right for you, we recommend [**HCP Boundary**](https://developer.hashicorp.com/boundary/tutorials/hcp-getting-started) because it eliminates deployment operations.

> Regardless of which server edition of Boundary you are using, all editions use the same Desktop and CLI clients, which you can download [here](https://developer.hashicorp.com/boundary/downloads).

Reviewer 2: ```suggestion
Regardless of which server edition of Boundary you use, all editions require the same Desktop and CLI clients, which you can download [here](https://developer.hashicorp.com/boundary/downloads).
```
